lib/N3Parser.js

```diff
@@ -775,8 +775,11 @@ N3Parser.prototype = {
   // ### `_error` emits an error message through the callback
   _error: function (message, token) {
     var err = new Error(message + ' on line ' + token.line + '.');
-    err.token = token;
-    err.line = token.line;
+    err.context = {
+      token: token,
+      line: token.line,
+      previousToken: this._lexer._prevToken,
```

Comment:
I'm not sure whether this line might become an issue. The token is passed as argument, where the previous token is fetched from the lexer directly. Is there a scenario where both are not 'in sync'?